exercises/proverb/example.js

```diff
@@ -0,0 +1,27 @@
+
+
+const lastArgIsOptions = (args) => {
+  const last = args[args.length - 1];
+  return typeof last === 'object';
+};
+
+const conclusion = (firstArg, qualifier = '') => `And all for the want of a ${qualifier}${firstArg}.`;
+
+const proverb = (...args) => {
+  let options = {};
+  if (lastArgIsOptions(args)) {
+    options = args.pop();
+  }
+
+  const allExceptLastArg = args.slice(0, -1);
+  const chainOfEvents = allExceptLastArg.map((arg, index) => `For want of a ${arg} the ${args[index + 1]} was lost.`);
+
+  const qualifier = options.qualifier ? `${options.qualifier} ` : options.qualifier;
+  chainOfEvents.push(conclusion(args[0], qualifier));
+
+  return chainOfEvents.join('\n');
+};
+
+
```

Comment:
nitpick: keep single blank lines for consitency. The ones on the top of this file can be removed.